Tests/MeiliSearchUnitTests/Utils.swift

```diff
@@ -6,3 +6,8 @@ import XCTest
 @testable import MeiliSearch
 
 public let TESTS_TIME_OUT = 10.0
+
+func decodeJSON<T: Decodable>(from string: String) throws -> T {
+  let data = Data(string.utf8)
+  return try Constants.customJSONDecoder.decode(T.self, from: data)
+}
```

Comment:
This isn't a necessary change for the sake of async/await switching, but utilising a common extension here allows us to remove more force unwraps increasing safety throughout tests, and shorter call site increases readability elsewhere.